lib/signs/bus.ex

```diff
@@ -168,22 +178,24 @@ defmodule Signs.Bus do
       |> filter_predictions(current_time, state)
       |> Enum.group_by(&{&1.stop_id, &1.route_id, &1.direction_id})
 
+    locals = %{
```

Comment:
I think part of the reason I'm not a fan of `locals` is because it implies "local variables", which are a specific thing, and this isn't quite that thing (you could say it's... _non_-local variables?). `context`, while similarly generic, doesn't have this problem. But don't feel like you have to change it on my preference alone 🙂